docs/StardustDocs/topics/toHTML.md

```diff
@@ -38,6 +38,7 @@ val df3 = df.sortByDesc { age }
 listOf(df1, df2, df3).fold(DataFrameHtmlData.tableDefinitions()) { acc, df -> acc + df.toHTML() }
 ```
 
+<dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Render.composeTables.html"/>
```

Comment:
Why do we need this line here?

Reply:
It's how we embed the table component into the documentation now